apps/fulltextsearch.sh

```diff
@@ -56,96 +57,57 @@ then
     deluser --group solr
 fi
 
-# Installing requirements
-echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections
-echo debconf shared/accepted-oracle-license-v1-1 seen true | sudo debconf-set-selections
-check_command apt install openjdk-8-jre -y
-check_command apt install apt-transport-https -y
+#Install docker (the ugly way)
+echo "Installing Docker CE..."
+curl -fsSL get.docker.com -o get-docker.sh
+bash get-docker.sh
 
-# Install Elastic
-check_command wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
-check_command echo "deb https://artifacts.elastic.co/packages/$ES_DEB_VERSION.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-"$ES_DEB_VERSION".x.list
-apt update -q4 & spinner_loading
-apt install elasticsearch=$ES_VERSION -y
-check_command /etc/init.d/elasticsearch start
-apt-mark hold elasticsearch
-
-# Enable on bootup
-sudo systemctl enable elasticsearch.service
-update-ca-certificates -f
-
-# Install ingest-attachment plugin
-if [ -d /usr/share/elasticsearch ]
-then
-    cd /usr/share/elasticsearch/bin
-    check_command ./elasticsearch-plugin install ingest-attachment
-fi
-
-# Check that ingest-attachment is properly installed
-if ! [ "$(curl -s 127.0.0.1:9200)" ]
-then
-msg_box "Installation failed!
-Please report this to $ISSUES"
-    exit 1
-fi
+#Prepare docker env
+nc_rores6x="ark74/nc_rores6.x:1.6.23_es6.3.2"
+rores6x_name="es6.3.2-rores_1.6.23"
+mkdir /opt/es/
+sysctl -w vm.max_map_count=262144
 
```

Comment:
Please move this to lib.sh. Thanks!

Reply:
mkdir and sysctl too?
Shouldn't they stay as they are actual commands executed?

If I'm getting this right when loaded lib.sh 
It will create the directory /opt/es/ and set vm.max_map... to 262144 regardless you want or not FTS in your system right?

Reply:
All the $vars and {functions} should live in lib.sh, it's just that. :)